add_to_app/MultipleFluttersIos/.gitignore

```diff
@@ -0,0 +1 @@
+MultipleFluttersIos.xcworkspace/xcuserdata
```

Comment:
be consistent in the folder name with the other samples (e.g. snake notation)

Reply:
I don't believe that is idiomatic rules for naming iOS projects.  I've never seen it in my 10 years of iOS development.

Reply:
![Screen Shot 2021-01-25 at 10 32 04 AM](https://user-images.githubusercontent.com/156888/105749659-920a9700-5ef8-11eb-980a-3d203ba42e93.png)

Sure I agree, but we don't want this to be the only folder with a different name either since there are many platforms involved.

Reply:
Okay, moved the folder, i can rename that folder fine and keep the project named idiomatically.  I got it twisted.